roles/jupyterhub/molecule/default/molecule.yml

```diff
@@ -0,0 +1,21 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: delegated
+# lint: |
+#   set -e
+#   yamllint .
+#   ansible-lint
+platforms:
+  - name: molecule-test
+provisioner:
+  name: ansible
+  inventory:
+    host_vars:
+      localhost:
+        ansible_python_interpreter: '{{ ansible_playbook_python }}'
+        kubeconfig: "{{ lookup('env', 'KUBECONFIG') }}"
+  env:
+    KUBECONFIG: /home/austin/.kube/config-molecule-test
+    K8S_AUTH_KUBECONFIG: /home/austin/.kube/config-molecule-test
```

Comment:
seems to have absolute paths and austin specific -- could be avoided ? may be at least via `~/` or `$HOME`?

Reply:
oops missed that one thanks